src/packages/xroad/debian/xroad-proxy.postinst

```diff
@@ -142,6 +142,27 @@ case "$1" in
  ;;
 esac
 
+#parameters:
+#1 file_path
+#2 old_section
+#3 old_key
+#4 new_section
+#5 new_key
+function migrate_conf_value {
+    MIGRATION_VALUE=$(crudini --get $1 $2 $3 2>/dev/null)
+    if [ ${MIGRATION_VALUE} ];
```

Comment:
word splitting happens if the parameter or migration_value has spaces. Use quotes around the parameters, e.g.  `if [ "{$MIGRATION_VALUE}" ];`